vincentvanbot/labels.py

```diff
@@ -0,0 +1,154 @@
+import os
+from os.path import join
+import io
+from google.cloud import vision, storage
+from vincentvanbot.params import IMAGES_PATH, LABELS_SELECTION, BUCKET_NAME
+from tqdm import tqdm
+
+tqdm.pandas(bar_format='{l_bar}{bar:30}{r_bar}{bar:-10b}')
+
+
+def get_labels_from_url(uri, max_results, proba_threshold=0.6, manual=True):
+    """Takes in uri (i.e. jpg link to an image). Returns dictionary having as keys the identified
+    labels, and as values their related proba.
+    In particular:
+    - only labels with proba > proba_threshold
+    - a maximum of max_results different labels
+    - only labels manually defined in LABELS_SELECTION (if manual)"""
+
+    # connect to google vision
+    client = vision.ImageAnnotatorClient()
+    image = vision.Image()
+
+    # get labels for input uri
+    image.source.image_uri = uri
+    response = client.label_detection(image=image, max_results=max_results)
+    labels = response.label_annotations
+
+    # create dict having as keys all labels, as values their related probas
+    # filter dict to only include labels where proba > proba_threshold
+    # filter dict to only include labels in LABELS_SELECTION if manual
+    labels_dict = {}
+    for label in labels:
+        if label.score > proba_threshold:
+            if manual:
+                if label.description in LABELS_SELECTION:
+                    labels_dict[label.description] = label.score
+            else:
+                labels_dict[label.description] = label.score
+    
+    return labels_dict
+
+def get_labels_from_local_path(path, max_results, proba_threshold=0.6, manual=True):
```

Comment:
A big chunk of code is duplicated from 'get_labels_from_url'. I would suggest to refactor this.